packages/orchestrator-ui-components/src/messages/en-US.json

```diff
@@ -37,6 +37,8 @@
         "start_workflow_title": "Start workflow"
     },
     "errors": {
+        "notAllowedWhenEngineIsNotRunningMessage": "This action is not allowed when the Workflow Engine is not running",
+        "notAllowedWhenEngineIsNotRunningTitle": "Workflow Engine is not running",
         "retrieve_stored_settings": "Something went wrong restoring your settings. Falling back to default settings",
```

Comment:
the mixing of camel case + snake case looks a bit weird.

Partially caused by form field names: that are python attributes on pydantic classes: so snake cased.